### DISCUSSION TITLE
Reacting on drag even with overlapping div

### DISCUSSION THREAD

**Discussion by piotrblasiak** [pmndrs/use-gesture#567] [Q&A]
I have a problem where I have a fullscreen div covering the node that I've used {...bind()} on, and even tho it's covering it - drag events are passed through (which is undesired). The covering div is a sibling and not a child, and I did try to set pointer-events: all on it which didn't make a difference. Is this by design and how can I fix it?

**Comment by dbismut** [comment]
Can you share a sandbox reproducing the issue? There's different ways of implementing what you described.

**Comment by piotrblasiak** [comment]
Hmm, nevermind - seems I can't reproduce the issue in a sandbox for some reason. Doesn't make sense but could be something else I guess.
